README.md

```diff
@@ -1 +1,42 @@
+- **Live link:**
+- **API documentation:** 
+1- [here(github)](https://docs.github.com/en/rest/guides/getting-started-with-the-rest-api)
+2-[here(brands phone api)](api-mobilespecs.azharimm.site/v2/brands)
+3- [here (phone api)](https://api-mobilespecs.azharimm.site/v2/brands/apple-phones-48?page=2)
+##:space_invader: The problem:
+Because of the epidemic that the world witnessed (Covid 19), which led to the deterioration of the economy and the closure of millions of different shops, which called for the establishment of electronic stores.
+##:wrench:description for the website:
+mobile store app. The idea of the project is a shop for selling mobile phones to all companies known in the world of mobile phones, where they move to the main menu of the site and add various devices of their types, issue and other additions, updates, deletions and various operations.
+## :clipboard: The wireframes:
+## :airplane: User Journey:
+For the first step, the user will visit the home page which it contains all the popular brands for mobiles. After choosing the brand he wants, a list of devices with the information for each device (the user maybe look for it) will appear. For the last step if he like the device he will click (add to cart) button, and enter the payment way he wants and the process will be done.
+## :rocket: User Stories:
+- As a user, you can enter the main page and choose the brand you want.
+- You can also see the latest versions of the phones.
+- As a user, when you click on any brand, it will show you all available phone versions from this brand.
+- As a user, if you like a phone, you can buy it by clicking on the shopping cart icon.
+- As a user, if you want to see your purchases, you can click on the word “My Cart” or press the menu button and choose “My Cart”.
+- You can also remove from the list of purchases by pressing the trash icon. 
+- As a user, you can find out all the details about the site and its owners by clicking on the about us iconز
+## :point_right:Steps for get the project on local machine:
+- Clone the repo by copy the link from button(code).
+- Open your terminal:
+  `git clone repo git link`
+  `cd name of repo` .
+- Use (code .) command to open VSC
+  than `go live server`
+- Watch the project and give us your opinion.
+## :beginner: Technologies used on this project:
+- HTML : build website structure.
+- CSS : adding style to the structure
+- JavaScript : data fetching, make the - - page dynamic
+- API: fetched data source
+- Local Storage : save the products that added to the cart
+##:crossed_swords: Team members:
+  ✧ [**Abdullah Al-Lahham**](https://github.com/moayed-nabahin):man_technologist:
+  ✧ [**Mostafa Balousha**](https://github.com/moayed-nabahin):man_technologist:
+  ✧ [**Moayed Al-Nabahin**](https://github.com/moayed-nabahin):man_technologist:
+  ✧ [**Nagham Abuwarda**](https://github.com/moayed-nabahin):woman_technologist:
```

Comment:
need format 
<img width="1076" alt="Screen Shot 2022-03-24 at 12 18 00 PM" src="https://user-images.githubusercontent.com/47992412/159894905-d2171141-d510-4ebc-9a18-f67412527c0c.png">


- Local Storage : save the products that added to the cart

## :crossed_swords: Team members:

  ✧ [**Abdullah Al-Lahham**](https://github.com/moayed-nabahin):man_technologist:
  
  ✧ [**Mostafa Balousha**](https://github.com/moayed-nabahin):man_technologist:
  
  ✧ [**Moayed Al-Nabahin**](https://github.com/moayed-nabahin):man_technologist:
  
  ✧ [**Nagham Abuwarda**](https://github.com/moayed-nabahin):woman_technologist: